src/main/viz/GenomeTrack.js

```diff
@@ -53,7 +53,7 @@ function renderGenome(ctx: DataCanvasRenderingContext2D,
         // We only push objects in the text case as it involves creating a
         // new object & can become a performance issue.
         // 0.5 = centered
-        ctx.fillText(letter, scale(1 + 0.5 + pos), height - 1);
+        ctx.fillText(letter, scale(1 + 0.5 + pos), height / 2);
```

Comment:
I'm on a Mac OS 10.11.2 using Chrome/54.0.2840.87. Regarding your screen capture, I saw the top cutoff as well on specific letters (G and T mostly), which is why using 13px may be the way to go - if we can't get the text to center vertically.